doc/libdnf5_plugins/actions.8.rst

```diff
@@ -0,0 +1,180 @@
+..
+    Copyright Contributors to the libdnf project.
+
+    This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+    Libdnf is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    Libdnf is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+
+.. _actions_plugin_ref-label:
+
+======================
+libdnf5 Actions Plugin
+======================
+
+Description
+===========
+
+The plugin allows to define actions to be executed during libdnf5 plugin callbacks.
+Each action is hooked to one specific callback. Actions for pre_transaction and
+post_transaction callbacks may define a (glob-like) filtering rule on the package
+NEVRA or package files, as well as whether the package is incomming or outgoing.
```

Comment:
```suggestion
This plugin allows defining actions to be executed during transactions through libdnf5
callback hooks. Each action is hooked to one specific callback. Actions for ``pre_transaction``
and ``post_transaction`` callbacks may define a (glob-like) filtering rule on the package
NEVRA or package files, as well as whether the package is incoming or outgoing.
```

Reply:
@Conan-Kudo Thanks. I used the suggested fixes with one change. I omitted "during transactions" from the first sentence. Some callback hooks are not related to a transaction and are called even if no transaction occurs.